hook-janitor/src/handlers/app.rs

```diff
@@ -0,0 +1,21 @@
+use axum::{routing, Router};
+use metrics_exporter_prometheus::PrometheusHandle;
+
+use hook_common::metrics;
+
+pub fn app(metrics: Option<PrometheusHandle>) -> Router {
+    Router::new()
```

Comment:
I'm confused by this http server. Who's going to be calling the janitor? Can't the janitor just track it's own metrics while running? Why does it need a server?

Reply:
Hmm, did I miss an easier way to expose the Prometheus HTTP metrics endpoint (that will be scraped by Victoria Metrics)?

Reply:
Ahh, is to scrap metrics. I guess I didn't like seeing pretty much the same server code as in producer. Maybe we can abstract this somehow? I will have to add the same thing in consumer probably

Reply:
Yeah, I think especially as part of a unified Rust workspace we should have a nice server wrapper that sets up metrics and everything.